dask/base.py

```diff
@@ -18,9 +17,47 @@
 __all__ = ("Base", "compute", "normalize_token", "tokenize", "visualize")
 
 
-class Base(object):
+class DaskIntf(object):
```

Comment:
The only reason I broke this off as its own object was to separate the expected interface from a base class with other common methods on it (`Base`). This seemed useful to me for documentation purposes, but does add yet-another-class. Happy to swap this out for just good docs.